department-of-veterans-affairs/caseflow
# Adding transaction for finalized / vacols upload

Connects #6546 

### Description
This PR adds a transaction block around setting a schedule period to finalized and uploading the algorithm results to VACOLS.

### Testing Plan
1. Confirm a VACOLS upload through the UI
1. Click the back button
1. Confirm through the rails console that finalized is set to true and the VACOLS updates have gone through
1. Click confirm again
1. Confirm the UI displays a success message but no additional hearings have been added to VACOLS
